docs/install/installing-istio.md

```diff
@@ -342,5 +339,5 @@ rm -rf istio-${ISTIO_VERSION}
 [2]:
   https://istio.io/docs/setup/kubernetes/additional-setup/sidecar-injection/#manual-sidecar-injection
 [3]: https://istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/
-[4]: https://istio.io/docs/tasks/security/mutual-tls/
+[4]: https://istio.io/pt-br/docs/concepts/security/#mutual-tls-authentication
```

Comment:
I don't think you need the `/pt-br/` language-specific docs (which doesn't seem to work anyway).

Reply:
Sorry my bad. Removed `pt-br`.